packages/app/src/hooks/useChartConfig.tsx

```diff
@@ -136,6 +142,10 @@ export function useQueriedChartConfig(
     refetchOnWindowFocus: false,
     ...options,
   });
+  useEffect(() => {
+    if (options?.onError && query.isError) options.onError(query.error);
+  }, [query.isError]);
+  return query;
```

Comment:
Just curious why you're using `useEffect` here?

Seems like this would be fine..or are you just trying to make sure that onError only gets called once?

```
  if (query.isError && options?.onError) {
    options.onError(query.error);
  }
```